### ISSUE TITLE
Package name showing as app name

### ISSUE THREAD

**Issue by overlord8** [forkgram/TelegramAndroid#207]
The pacakage name of this app 'org.telegram.messenger.ApplicationLoaderImpl' showing as app name both in app drawer and in settings -> manage apps.
please fix this issue

**Comment by formula-spectre** [comment]
bump, same here

**Comment by suzver** [comment]
version from F-Droid? https://github.com/forkgram/TelegramAndroid/issues/160

**Comment by Uz4ever** [comment]
Same here
![Screenshot_20231117-092947_Impostazioni](https://github.com/forkgram/TelegramAndroid/assets/37186923/5eb57f11-95f4-4e6f-93c9-afe14a23fafc)
![Screenshot_20231117-093012_Sistema Android](https://github.com/forkgram/TelegramAndroid/assets/37186923/711c8670-447a-40f0-ab19-22b05b6722fa)

**Comment by formula-spectre** [comment]
as someone else said, just download the version from github, it has an automatic updater integrated

**Comment by et-ness** [comment]
Duplicate #110 please close, there are too many issues always with the same request.
Why don't people search before opening?

> as someone else said, just download the version from github, it has an automatic updater integrated

Sure, but there are 2 ways. Remove ForkGram from Fdroid or fix it. There is also a PR, but still don't merged.
I think is wrong keep forkgram on fdroid with that isuee.

**Comment by 0slav** [comment]
> Sure, but there are 2 ways. Remove ForkGram from Fdroid or fix it. There is also a PR, but still don't merged. I think is wrong keep forkgram on fdroid with that isuee.

No. I think it's much better to keep forkgram on fdroid than removing it over a small cosmetic issue or merging low-priority PRs without proper testing.

**Comment by et-ness** [comment]
> low-priority PRs without proper testing.

what are you talking about? It is simply a label to be inserted into an xml file. The same label is inserted when is compiled on github.

> I think it's much better to keep forkgram on fdroid

Is better to remove, forkgram already check updates automatically on github.

**Comment by 0slav** [comment]
Yes it's a small change, but just a cosmetic one and if the dev can't be bothered to fix it, that's fine. Feel free to install the github release version, I'll rather keep using the fdroid one.

Updating trough fdroid is more convenient and it provides some level of added trust due to automated tests and vulnerability warnings for known unfixed vulnerabilities.

**Comment by nach0** [comment]
Just fix the F-Droid version! This is a known problem for looong time and what I've read implies there's a fix ready to be merged. There's nothing to discuss: apply the fix and get rid of the stupid nonsense app name

**Comment by Zeit08** [comment]
Agree, how do we promote the change to be implemented?

**Comment by ghost** [comment]
@23rd , dear devs, could you fix this issue please?
This problem has been persisting since I started using this program ~1.5 years ago and it is still here,
and given how much visible it is, it's hard to understand how it managed to remain for so long.

**Comment by ghost** [comment]
> closed this as [completed](https://github.com/forkgram/TelegramAndroid/issues?q=is%3Aissue+is%3Aclosed+archived%3Afalse+reason%3Acompleted) [3 hours ago](https://github.com/forkgram/TelegramAndroid/issues/207#event-11616204307)

Has it already been fixed?

**Comment by ghost** [comment]
Just checked, the bug is still there
@overlord8 could you re-open the issue?

**Comment by ghost** [comment]
As of July 2024, the bug is still present.

Again, why was the issue closed? @overlord8 could you re-open the issue?

**Comment by overlord8** [comment]
I don't use forkgram right now. switched to nekox broken.
________________________________

**Comment by ghost** [comment]
@overlord8 , ok but could you re-open the issue?

**Comment by overlord8** [comment]
> @overlord8 , ok but could you re-open the issue?

done sur.

**Comment by bt4ibwem8** [comment]
@overlord8 
Why did you reopen it? It is duplicate of #110

**Comment by Unicorn-42** [comment]
Could this issue be fixed?

This should be a simple 1-line fix, why does this issue still persist? it's been years!

**Comment by 23rd** [comment]
That was fixed.
